common/web/keyboard-processor/build.sh

```diff
@@ -45,21 +45,26 @@ function do_build() {
   tsc --build "$THIS_SCRIPT_PATH/tsconfig.all.json"
 
   # Base product - the main keyboard processor
-  $BUNDLE_CMD    "${KEYMAN_ROOT}/common/web/keyboard-processor/build/obj/index.js" \
+  ${BUNDLE_CMD}    "${KEYMAN_ROOT}/common/web/keyboard-processor/build/obj/src/index.js" \
```

Comment:
Why the change in path here?

Reply:
Also, just to be clear - be _very_ careful when changing the build-output paths like this - it has a tendency to have unwanted side effects in the end-result sourcemap files, often causing the pathing structure shown when debugging via Chrome to become very messy and thus more difficult to utilize.

Reply:
Makes sense. Done.